index.html

```diff
@@ -9,7 +9,7 @@
 
 
 
-    <script src="app.js"></script>
+    
 </head>
 <body>
     <div class="container">
```

Comment:
There should be no HTML here anymore. The only thing that should exist is a `div` which the backbone app renders to. The rest should all be done with templates for views.